src/websockets/TimerService/utility.js

```diff
@@ -0,0 +1,63 @@
+import { TIMER_UPDATES_CHANNEL } from 'websockets/api';
+
+const logger = require('../../startup/logger');
+
+export const calculateTotalSecondsBasedOnStartingTime = ({ currentTimeInSeconds, startedAtInSeconds, seconds }) => currentTimeInSeconds
+          - startedAtInSeconds
+  + seconds;
+
+export const getTimerFromRedis = async ({
+  userId,
+  redisClient,
+} = {}) => {
+  try {
+    return JSON.parse(await redisClient.get(userId) ?? '{}');
```

Comment:
If userId == null, do we want to do get with an empty object?

It seems like throwing an error is good here.

Reply:
I mean from a front-end perspective nothing would change.